templates/graphana/bucket_panel_template.json

```diff
@@ -0,0 +1,80 @@
+{
+  "aliasColors": {},
+  "bars": true,
+  "dashLength": 10,
+  "dashes": false,
+  "datasource": "account-cloudwatch",
+  "fill": 1,
+  "fillGradient": 0,
+  "gridPos": {
+    "h": 8,
+    "w": 9,
+    "x": 5,
+    "y": 0
+  },
+  "id": 4,
+  "legend": {
+    "avg": false,
+    "current": true,
```

Comment:
@chmreid @DailyDreaming this will show a stacked bar char that sums the stacks, but will provide the separate series information within the legend

Reply:
thanks @amarjandu